README.md

```diff
@@ -0,0 +1,129 @@
+# influxdb
+
+influxdb provides an async Rust interface to an [InfluxData][] database.
```

Comment:
Should check the crate docs in the other PR to steal my unified wording of influx / influxdata / influxdb.